### ISSUE TITLE
[BUG] Run a single test file but run twice

### ISSUE THREAD

**Issue by CodingProgrammer** [microsoft/playwright#23428]
### System info
- Playwright Version: [v1.33.0]
- Operating System: [Windows 
- Browser: [Chromium]
- Other info:
I have the folder tree like this: sanity/settings/sanity-settings-sidebar-behavior.spec.ts

**Steps**
- Run the test with the command:  `npm run pwa-chat-test sanity-settings-sidebar-behavior.spec.ts`

**Expected**

[Run sanity-settings-sidebar-behavior.spec.ts file only once]

**Actual**

[Run the test file twice. Once in the folder sanity/settings/ and other in the folder /settings]
<img width="610" alt="image" src="https://github.com/microsoft/playwright/assets/22937096/06f5181b-562a-45a9-8bd9-146ab276dc24">

**Comment by mxschmitt** [comment]
Do you have a reproduction which you can share? Maybe it is defined in multiple projects! You can use:

```
npm run pwa-chat-test -- sanity-settings-sidebar-behavior.spec.ts --list
```

to find out which tests will run before they actually run.

**Comment by aslushnikov** [comment]
Looks like the discussion has stalled here - closing this.

Please do not hesitate to file a new issue with a repro if you think we still can be helpful.
